.gitignore

```diff
@@ -1,5 +1,6 @@
 TODO
 coverage
+dist
```

Comment:
yeah it's annoying, wish there was a better way to provide the browser builds, maybe I should post them somewhere instead of checking them in.